Day-2/provider.tf

```diff
@@ -0,0 +1,15 @@
+# Terraform block
+terraform {
+  required_version = ">=1.5.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">=5.5.0"
+    }
+  }
+}
+
+# provider block
+provider "aws" {
+  region = var.region
+}
```

Comment:
could you please, add a newline at the end of file.

Reply:
added new lines at the end